### PR TITLE
Character controller: change solve_character_collision_impulses to take multiple CharacterCollision

### PR DESCRIPTION
**Pull request by Vrixyz**
https://github.com/dimforge/rapier.rs/pull/104 will have to adapt to this modification

### CODE REVIEW

File: src/control/character_controller.rs [hunk 1]
```diff
@@ -782,11 +782,40 @@ impl KinematicCharacterController {
...
+                filter,
+            );
+        }
+    }
+
+    /// For the given collision between a character and its environment, this method will apply
+    /// impulses to the rigid-bodies surrounding the character shape at the time of the collision.
+    /// Note that the impulse calculation is only approximate as it is not based on a global
+    /// constraints resolution scheme.
+    fn solve_character_collision_impulse(
```
Comment (sebcrozet) [inline src/control/character_controller.rs:818]: Just to make it extra obvious it solves a single collision:
```suggestion
    fn solve_single_character_collision_impulse(
```

**Review by sebcrozet — APPROVED** [review]
Thanks!

File: src/control/character_controller.rs (hunk 1 above)
Comment (Vrixyz) [reply src/control/character_controller.rs:818]: Notice I changed its visibility to private, is that ok ? @sebcrozet

Comment (sebcrozet) [reply src/control/character_controller.rs:818]: Yes, making it private is good.

**Pull request merged**